packages/slice-machine/src/apiClient.ts

```diff
@@ -27,6 +27,7 @@ export const getState = async (): Promise<ServerState> => {
   // transform the data to something SM recognizes.
   const state: ServerState = {
     ...rawState,
+    // @ts-expect-error TS(2322) FIXME: Type '{ framework: Frameworks; mockConfig: CustomT... Remove this comment to see the full error message
```

Comment:
It's real. The type in the manager comes from a library @antfu/ni, which contains a few more options than the one in SM UI. At some point these should be aligned, but I've seen a lot of TODO's around getState so didn't want to be too preemptive with the change.

Maybe @angeloashmore can provide more clarity at some point since those TODOs seem to be from him 🙂